src/parser/mod.rs

```diff
@@ -8349,6 +8349,13 @@ impl<'a> Parser<'a> {
     pub fn parse_object_name(&mut self, in_table_clause: bool) -> Result<ObjectName, ParserError> {
         let mut idents = vec![];
         loop {
+            if self.dialect.supports_object_name_double_dot_notation()
+                && idents.len() == 1
+                && self.peek_token() == Token::Period
+            {
+                self.next_token();
+                idents.push(Ident::new(""));
```

Comment:
Oh is this line correct, I imagined we would push `Ident::new(".")`?

Reply:
Ah I realised this is empty due to the display adding the dot on its behalf. Could we add a comment what this line does? since its not obvious otherwise

Reply:
Yes, I'm also waiting on #1539 to merge and then I will change this to something like `DefaultSchema` instead.